Develop an SQL schema for SOPs

```diff
@@ -450,39 +685,63 @@ CREATE SEQUENCE public."STUDY_DESIGN_DESCRIPTORS_Design_Descriptors_ID_seq"
     CACHE 1;
 
 
-ALTER TABLE public."STUDY_DESIGN_DESCRIPTORS_Design_Descriptors_ID_seq" OWNER TO postgres;
+ALTER TABLE public."MzTab-M_Small_Molecule_Feature_(SMF)_Section_SMF_ID_seq" OWNER TO postgres;
 
 --
--- TOC entry 2958 (class 0 OID 0)
--- Dependencies: 216
--- Name: STUDY_DESIGN_DESCRIPTORS_Design_Descriptors_ID_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+-- TOC entry 3457 (class 0 OID 0)
+-- Dependencies: 239
+-- Name: MzTab-M_Small_Molecule_Feature_(SMF)_Section_SMF_ID_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
 --
 
-ALTER SEQUENCE public."STUDY_DESIGN_DESCRIPTORS_Design_Descriptors_ID_seq" OWNED BY public."STUDY_DESIGN_DESCRIPTORS"."Design_Descriptors_ID";
+ALTER SEQUENCE public."MzTab-M_Small_Molecule_Feature_(SMF)_Section_SMF_ID_seq" OWNED BY public."MzTab-M_Small_Molecule_Feature_(SMF)_Section"."SMF_ID";
 
 
 --
--- TOC entry 219 (class 1259 OID 17562)
--- Name: STUDY_FACTORS; Type: TABLE; Schema: public; Owner: postgres
+-- TOC entry 237 (class 1259 OID 18396)
+-- Name: MzTab-M_small_molecule_section; Type: TABLE; Schema: public; Owner: postgres
 --
 
-CREATE TABLE public."STUDY_FACTORS" (
-    "Study_Factors_ID" integer NOT NULL,
-    "Factor_Name" text,
-    "Factor_Type" text,
-    "Factor_Type_Term_Accession_Number" text,
-    "Factor_Type_Term_Source_REF" text
+CREATE TABLE public."MzTab-M_small_molecule_section" (
+    "SML_ID" integer NOT NULL,
+    "SMF_ID_REFS_number" integer,
```

Comment:
What is the difference between the SMF_ID_REFS_number and SMF_ID_REFS columns?

Reply:
if ambiguity remains, then several values may be provided in SMF_ID_REFS column, so i created SMF_ID_REFS_number column in case they are 2 values in SMF_ID_REFS cols.